sdk/schemaregistry/azure-data-schemaregistry/src/main/java/com/azure/data/schemaregistry/implementation/models/SchemasGetByIdHeaders.java

```diff
@@ -60,13 +60,19 @@ public final class SchemasGetByIdHeaders {
      * @param rawHeaders The raw HttpHeaders that will be used to create the property values.
      */
     public SchemasGetByIdHeaders(HttpHeaders rawHeaders) {
-        this.schemaVersion = Integer.parseInt(rawHeaders.getValue("Schema-Version"));
+        String schemaVersion = rawHeaders.getValue("Schema-Version");
+        if (schemaVersion != null) {
```

Comment:
Behavioural breaking change, as previously NPE would be thrown.
Plus the if check is not consistent for all values being read below.